npm/cmd/start.go

```diff
@@ -110,7 +111,7 @@ func start(config npmconfig.Config) error {
 
 	var dp dataplane.GenericDataplane
 	if config.Toggles.EnableV2NPM {
-		dp, err = dataplane.NewDataPlane(npm.GetNodeName(), common.NewIOShim())
+		dp, err = dataplane.NewDataPlane(npm.GetNodeName(), common.NewIOShim(), policies.IPSetAndNoRebootConfig)
```

Comment:
We might have a lot more configs coming our way our dp, policy and ipsets mgrs, so lets create a new Dataplane config ? which inherits both policyMgrCfg and ipsetMgrCfg, that way we will have to send in only one cfg option ?